plugins/woocommerce/tests/php/includes/class-wc-order-item-fee-test.php

```
@@ -0,0 +1,296 @@
+	 * @testdox set_taxes filter allows plugins to customize legacy tax conversion.
+	 */
+	public function test_set_taxes_legacy_conversion_filter() {
+		// Create an order - the filter is only called when an order context exists.
+		$order = WC_Helper_Order::create_order();
+		$order->save();
+
+		$item = new WC_Order_Item_Fee();
+		$item->set_order_id( $order->get_id() );
+
+		// Add a filter to customize the conversion.
+		$filter_callback = function ( $converted, $value, $order_item ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$converted'. (undefined)

(UnusedFormalParameter)